src/utils/mathUtils.ts

```diff
@@ -61,18 +61,11 @@ export module Utils {
     }
 
     /**
-     * Returns true **only** if x is NaN
-     */
-    export function isNaN(n: any) {
-      return n !== n;
-    }
-
-    /**
      * Returns true if the argument is a number, which is not NaN
      * Numbers represented as strings do not pass this function
      */
     export function isValidNumber(n: any) {
-      return typeof n === "number" && !Plottable.Utils.Math.isNaN(n) && isFinite(n);
+      return typeof n === "number" && n === n && isFinite(n);
```

Comment:
Are you suggesting that we should use `isNaN()` in those situations rather than removing this function?  Note that I saw 0 usage of this function in the source code.  It is only used in the tests, meaning that at best it should be a function in the utils, but since it's used in this one place and `isNaN` is so short, I decided to just inline the definition